ui/packages/shared/client/src/parca/query/v1alpha1/query.ts

```diff
@@ -602,6 +608,75 @@ export interface FlamegraphNodeMeta {
      */
     line?: Line;
 }
+/**
+ * CallgraphNode represents a node in the graph
+ *
+ * @generated from protobuf message parca.query.v1alpha1.CallgraphNode
+ */
+export interface CallgraphNode {
```

Comment:
Do we plan to have some other information for each node in addition to the name and id? At a minimum, it'd be helpful to have the information required by the flamegraph tooltip so we could potentially reuse the contents of that tooltip component. For example, a "cumulative"/total value, the sample unit, and meta info like the function name and location address?

Reply:
Done, added this.